Gadget.Notifications/Controllers/NotificationsController.cs

```diff
@@ -42,5 +43,17 @@ public NotificationsController(INotificationsService notificationsService)
             var webhooks = await _notificationsService.GetWebhooks(agentName, serviceName, cancellationToken);
             return Ok(webhooks);
         }
+
+        private NotifierType ParseNotifierType(string notifierTypeName)
```

Comment:
jak juz nie testujesz, to usuwamy

Reply:
ten parser jest mi potrzebny, czy po prosty dać (NotifierType)createWebhook.NotifierType  ?

Reply:
ale do czego potrzebny

Reply:
dajesz enuma jako int w requestcie

Reply:
nie parsujesz stringa na enuma,  po to maszendpoint types, zeby sobie pobrac info o tym enumie